digits/job.py

```diff
@@ -202,3 +211,44 @@ def disk_size_fmt(self):
         size = fs.get_tree_size(self._dir)
         return sizeof_fmt(size)
 
+    def get_tasks_recursively(self):
+        tasks = []
+        for task in self.tasks:
+            if task:
+                tasks.append(task)
+                tasks += task.get_tasks_recursively()
+        return tasks
```

Comment:
Yes, all tasks are in the `job.tasks` list. The `task.parents` field is optional, and may describe dependencies between tasks. The tasks are not necessarily a fully-connected graph. We can change this behavior if you have a good reason for it.